src/views/profiles/ViewProfile.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import ProfileView from '../../components/profiles/ViewProfile';
+import { fetchProfile, editProfile } from '../../redux/actions/profile.actions';
+
+class ViewProfile extends Component {
+  state = {};
+
+  componentDidMount() {
+    const { fetchProfile: Profile } = this.props;
+    Profile(this.state);
+  }
+
+  render() {
+    const { profile: { username, bio, img_url, first_name, last_name } } = this.props;
+    return (
+      <ProfileView
+        username={username}
+        image={img_url}
+        bio={bio}
+        first_name={first_name}
+        last_name={last_name}
+        handleCick={this.handleCick}
```

Comment:
handleCick is not defined. Neither are you using it in the ProfileView Component

Reply:
Done